### DISCUSSION TITLE
Riverpod Migration Tool?

### DISCUSSION THREAD

**Discussion by TJMusiitwa** [rrousselGit/riverpod#1463] [Q&A] [answered]
I have looked at the repository issues and have noticed that there are a few issues with the tag `migration tool`. 
I am interested in this because it seems like it would be a great tool to use in a large project that has providers everywhere and would like to transition to using riverpod. 
That being the case I was wondering @rrousselGit if this is something coming later on to Riverpod or perhaps should one just take on the task of manually migrating  _providers_ with the similar _riverpod-providers_
**In overall I would just like to know what the riverpod migrate is**
Thank you

**Comment by TimWhiting** [comment, marked as answer]
Riverpod's migration tool was created from pre 1.0.0 riverpod to riverpod 1.0.0. Unfortunately there ended up being some issues with it, due to the complexity of the migration. I was the primary developer on that, but unfortunately am very busy with school and other priorities. Remi and I have talked about providing a migration from package:provider Providers to riverpod providers. While I think it is possible, it is also more work than I can commit to right now.

**Reply by TJMusiitwa** [reply]
Oh thank you for your contribution @TimWhiting. Understandably, some other commitments are taking some time but I am hopeful when you are able to come back to this it should proceed along well. 
For now I think most of us can just be able to manually convert providers without much issue.
Thanks again
